lib/fake_ifdh.py

```diff
@@ -57,6 +57,19 @@ def getRole(role_override: Optional[str] = None, verbose: int = 0) -> str:
     if role_override:
         return role_override
 
+    # if we have a default role pushed with a vault token, or $HOME/.jobsub_default... use that
+    uid = os.getuid()
+    group = os.environ["GROUP"]
+
+    for prefix in ["/tmp/", f"{os.environ['HOME']}/."]:
```

Comment:
I was looking at this, and for the sake of consistency, I think the filename should be the same, whichever directory it's in.  So either `.jobsub_default_role_{group}_{uid}` or `jobsub_default_role_{group}_{uid}`.  Otherwise, people might get confused.  What do you think?

Reply:
That's fine.  or should we make it $HOME/.config/jobsub_default_role_{group}_{uid} ?  Then it's not clogging up the home directory, and its spelled the same?

Reply:
That's a good idea.